### ISSUE TITLE
Refer to legacy liquidation portal from dai bidding flow

### ISSUE THREAD

**Issue by LukSteib** [sidestream-tech/unified-auctions-ui#248]

**Comment by valiafetisov** [comment]
Yes, I like the second proposal with the link inside "Different ways to bid" block

This issue can also be used to update our list of tools on the main page:
- to change title of our tool (or show as a separate block under another title, but that's confusing)
- to remove primary color from the `participate` button of the "Liquidations platform":

<img width="650" alt="Screenshot 2022-05-05 at 17 38 50" src="https://user-images.githubusercontent.com/3393626/166960624-991c9d70-2a3c-4d96-a57e-e837ee5c5bf8.png">

**Comment by LukSteib** [comment]
> This issue can also be used to update our list of tools on the main page:

Yeah created an issue for that as well already. But we can cover it here https://github.com/sidestream-tech/unified-auctions-ui/issues/246
